### PR TITLE
Use better-files everywhere.

### PR DESCRIPTION
**Pull request by danxmoran**
### Description

In #217 I added better-files on @pshapiro4broad's suggestion, but I didn't go all the way down the rabbit hole because I wasn't sure how it'd play with the google NIO adapter and I wanted to get the release rolling. This PR finishes the job.

----

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: Ensure that you have added or updated tests
- [ ] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=114531509).
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
* Review cycle:
  * Team may comment on PR at will
  * **Reviewer assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to reviewer** for further feedback
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Move the JIRA issue to QA once this checklist is completed

### CODE REVIEW

File: clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioParsers.scala [hunk 1]
```diff
@@ -1,16 +1,16 @@
 package org.broadinstitute.clio.client.commands
 
 import java.net.URI
-import java.nio.file.{Path, Paths}
+import java.time.OffsetDateTime
```
Comment (als364) [inline clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioParsers.scala:4]: How was this working before if this and `java.util.UUID` were not imported?

File: clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/WgsCramTests.scala [hunk 2]
```diff
@@ -605,52 +591,45 @@ trait WgsCramTests { self: BaseIntegrationSpec =>
...
+    )
 
-    val _ = Seq(
-      (cramPath, cramContents),
-      (craiPath, craiContents),
-      (metrics1Path, metrics1Contents),
-      (metrics2Path, metrics2Contents)
-    ).map {
-      case (path, contents) => Files.write(path, contents.getBytes)
+    val _ = if (!testNonExistingFile) {
```
Comment (pshapiro4broad) [inline clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/WgsCramTests.scala:622]: Here's a case where `unless` might actually be helpful

**Review by pshapiro4broad — APPROVED** [review]
👍 👍

**Review by als364 — APPROVED** [review]

File: clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioParsers.scala (hunk 1 above)
Comment (danxmoran) [reply clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioParsers.scala:4]: It was imported below, I snuck an optimize-imports in.

**Pull request merged**